tests/commands/test_node.py

```diff
@@ -0,0 +1,48 @@
+"""Tests for node command."""
+import logging
+from unittest.mock import patch
+
+import pytest
+
+from nile.core.node import node
+
+HOSTS = ["127.0.0.1", "goerli"]
+PORTS = ["5050", "5001"]
+LITE = "--lite-mode"
+
+
+@pytest.fixture(autouse=True)
+def tmp_working_dir(monkeypatch, tmp_path):
+    monkeypatch.chdir(tmp_path)
+    return tmp_path
+
+
+@pytest.mark.parametrize(
+    "args, host, port, mode",
+    [
+        ([], HOSTS[0], PORTS[0], None),
+        ([HOSTS[1], PORTS[1]], HOSTS[1], PORTS[1], None),
+        ([HOSTS[1], PORTS[1], LITE], HOSTS[1], PORTS[1], LITE),
```

Comment:
i’m not sure why these two lines use `HOSTS[1]`, `PORTS[1]`, and `LITE` as args. i guess they’re just placeholder values but if that's the case then it causes a bit of confusion. or do they serve a different purpose?

Reply:
It's to make sure that when these args are explicitly passed, it doesn't cause any issues with the command (like it did with `--account_contract` and `disable_hint_validation ` in #150). I'll refactor to make it less confusing